sdks/python/test-suites/direct/py35/build.gradle

```diff
@@ -34,6 +34,7 @@ task postCommitIT(dependsOn: 'installGcpTest') {
         "apache_beam.io.gcp.bigquery_io_read_it_test",
         "apache_beam.io.gcp.bigquery_read_it_test",
         "apache_beam.io.gcp.bigquery_write_it_test",
+        "apache_beam.io.gcp.bigtableio_it_test",
```

Comment:
there is also https://github.com/apache/beam/blob/acb1f561cc0803367010dc025d13a41b50611f7d/sdks/python/build.gradle#L108 . The location of Py27 suite is currently not consistent with 3.5-3.7.

Reply:
Done.